practiceCourseGenerator/practiceCourseGenerator.py

```diff
@@ -0,0 +1,233 @@
+'''
+
+@author: Kyle Dove
+
+Purpose: Download users list from Google Drive. Generate CSV files in 
+directory. Zip directory to be used to create Canvas practice courses.
+
+DATE           Author               Description   
+===========    ===============      ============================================
+Mar 31 2015    Kyle Dove			TLUNIZIN-470
+									Created.
+
+'''
+
+import logging
+import time
+import csv
+import requests
+import json
+import os
+import zipfile
+from apiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+from apiclient import errors
+from apiclient import http
+
+def download_file(service, drive_file):
+    download_url = drive_file['exportLinks']['text/csv']
+    logger.info('DownloadUrl: ' + download_url)
+    if download_url:
+            resp, content = service._http.request(download_url)
+            if resp.status == 200:
+                    logger.info('Status: %s' % resp)
+                    title = drive_file.get('title')
+                    path = './input.csv'
+                    file = open(path, 'wb')
+                    file.write(content)
+            else:
+                    logger.info('An error occurred: %s' % resp)
+                    return None
+    else:
+            # The file doesn't have any content stored on Drive.
+            return None
+
+def zipdir(path, zip):
+    for root, dirs, files in os.walk(path):
+        for file in files:
+            zip.write(os.path.join(root, file))
+
+#setup log
```

Comment:
I think the main body of this program has gotten a little too big and some code is repeated.  I'd suggest moving the setup and initialization code to function(s).  The Google Drive setup code should definitely be in its own function, so it's easy to see where it begins and ends.